lib/src/mock_platform_adapter.dart

```diff
@@ -0,0 +1 @@
+
```

Comment:
it would definitely be nice to see how the mock adaptor fits into this, but I'm fine with handling this in 2 pieces if you want